include/gsl/span

```diff
@@ -70,9 +71,9 @@ namespace gsl
 {
 
 // [views.constants], constants
-constexpr const std::ptrdiff_t dynamic_extent = -1;
+constexpr const std::size_t dynamic_extent = static_cast<std::size_t>(-1);
```

Comment:
Pre-existing: The lack of `inline constexpr` in C++17 mode is a minor difference from the Standard specification. It is unlikely to matter, but it is theoretically detectable if someone takes this variable's address.